doc/omm_rosetta.rst

```diff
@@ -0,0 +1,82 @@
+Seeding OpenMM Simulations with PyRosetta and ParmEd
+====================================================
+
+In this example, we'll go over starting a small protein
+(dodeca-alanine) simulation from scratch, using PyRosetta and OpenMM.
+
+Generate pose from primary structure
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To start, let's create a PyRosetta :class:`Pose`
+directly from a sequence of twelve alanines::
+
+    from rosetta import init, pose_from_sequence
+
+    init()
+
+    seq = 12*'A'
+    pose = pose_from_sequence(seq)
+
+
+Mutate a residue
+~~~~~~~~~~~~~~~~
+
+Want to mutate the fifth residue to a lysine? PyRosetta
+makes it as easy as::
+
+    from toolbox import mutate_residue
```

Comment:
It's really `from toolbox import mutate_residue`?  `toolbox` isn't part of the `rosetta` namespace?  That's almost as crazy as using the name `chemistry` as a package :).